data-access/Models/TextFragment.cs

```diff
@@ -14,6 +14,7 @@ public class TextFragmentData
     {
         public string TextFragmentName { get; set; }
         public uint TextFragmentId { get; set; }
+        public uint EditionEditor { get; set; }
```

Comment:
Shouldn't this be EditionEditorId?

Reply:
Yes, it should be.  I'll fix that in the next pull.  Nice catch.